.golangci.yml

```diff
@@ -13,17 +13,30 @@ linters:
     - misspell
     - unused
     - whitespace
+  disable:
+    - errcheck
 
 linters-settings:
   goimports:
     local-prefixes: github.com/openservicemesh/osm
 
 issues:
-  new: true
   exclude-rules:
     # Ignore error for ginkgo and gomega dot imports
     - linters:
         - golint
       source: ". \"github.com/onsi/(ginkgo|gomega)\""
       text: "dot imports"
+    # ignore unused linters on keyvault as this code isn't being used
+    # TODO: disable this rule once keyvault is integrated
+    - path: pkg/certificate/providers/keyvault/
+      linters: 
+        - unused
+        - deadcode
+    # Exclude staticcheck messages for deprecated function, variable or constant
+    # This causes issues with package github.com/golang/protobuf/proto
+    - linters:
+        - staticcheck
+      text: "SA1019:"
+  exclude-use-default: false
```

Comment:
This flag will ensure that the `golint` check locally and on the CI return the same result